app/main/views/briefs.py

```diff
@@ -360,6 +360,43 @@ def award_brief_details(brief_id, brief_response_id):
     return single_result_response(RESOURCE_NAME, brief), 200
 
 
+@main.route('/briefs/<int:brief_id>/award/<int:brief_response_id>/contract-details', methods=['DELETE'])
+def un_award_brief_details(brief_id, brief_response_id):
+    updater_json = validate_and_return_updater_request()
+
+    brief_response = BriefResponse.query.filter(
+        BriefResponse.brief_id == brief_id,
+        BriefResponse.id == brief_response_id
+    ).first_or_404()
+
+    if brief_response.status != 'awarded':
+        abort(400, "Can only un-award an awarded brief response")
+
+    # Set status back to pending-awarded
+    brief_response.awarded_at = None
+    brief_response.award_details = {'pending': True}
+
+    audit_event = AuditEvent(
+        audit_type=AuditTypes.update_brief_response,
+        user=updater_json['updated_by'],
+        data={
+            'briefId': brief_id,
+            'briefResponseId': brief_response_id,
+            'briefResponseAwardedValue': True,
```

Comment:
I'm not sure what this piece of information is for, could you explain for me?

Reply:
Hmm, I'm not sure about this either. I think the idea was that it was a similar action to this https://github.com/alphagov/digitalmarketplace-api/blob/acc24c649d316e37059c51e336c58978f2ca028f/app/main/views/briefs.py#L299 which sets the status to pending, but I agree it is confusing. Do you have any further thoughts @bjgill?

Reply:
Yes. I think we're just doing it to match the method that sets the brief response to `pending-awarded`, which is also what we're doing here.

But happy to remove `briefResponseAwardedValue` if you think it's likely to cause confusion?

Reply:
Hmm, I guess I'd just like to know what it means, if anything 😅

Reply:
Searching our code for the key name does not elucidate anything.

Reply:
I'm not sure it means anything :shrug:. It's used in `/briefs/<int:brief_id>/award` when a brief response is moved to `pending-awarded`, so we're just doing it for consistency.

Reply:
I guess it doesn't matter too much.